inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/config/ControllerExceptionHandler.java

```diff
@@ -41,18 +42,19 @@
 /**
  * Handler of controller exception.
  */
-@Slf4j
 @RestControllerAdvice
 public class ControllerExceptionHandler {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ControllerExceptionHandler.class);
```

Comment:
Why not use @Slf4j?

Reply:
I saw other module use `LOGGER`, but i don't notice **manager-web** use `@slf4j`. i have correct it.

Reply:
The new code is more @Slf4j

Reply:
right, i have correct it.